frontend/src/employee-frontend/components/unit/unit-reservations/ReservationsTable.tsx

```diff
@@ -113,9 +137,8 @@ const CustomTh = styled(Th)<{ shrink?: boolean }>`
     `}
 `
 
-const DateTh = styled(CustomTh)<{ faded: boolean; highlight: boolean }>`
+const DateTh = styled(CustomTh)<{ faded: boolean }>`
   ${(p) => p.faded && `color: ${colors.grayscale.g35};`}
-  ${(p) => p.highlight && `border-bottom: 2px solid ${colors.main.m3};`}
```

Comment:
Is there a reason you took this highlight out? It's there in UI specs.

Reply:
Ah it seems to be included in one figma view and but not in the other :) I was looking at the other...

Reply:
Then I don't know if it should be taken out or not 😄